tests/e2e-leg-10/sandbox-basic/setup-vdb/base/setup-vdb.yaml

```diff
@@ -23,7 +23,7 @@ spec:
   initPolicy: CreateSkipPackageInstall
   communal: {}
   local:
-    requestSize: 250Mi
+    requestSize: 270Mi
```

Comment:
Will 20Mi make any difference? We shouldn't make this change.

Reply:
I noticed the vdb received low disk space event when I ran the test locally. When free disk space is less than 10MB, that event will be sent.

Reply:
Then give it more like 500Mi.

Reply:
500Mi is great for ci server. But when we run the test locally, it may be a challenge if the free disk space is not large enough. When parallel is set to 2, there will 20 pods at most at the same time. How about 300Mi?